nifi-extension-bundles/nifi-groovyx-bundle/nifi-groovyx-processors/src/main/java/org/apache/nifi/processors/groovyx/ExecuteGroovyScript.java

```diff
@@ -483,7 +483,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession _sessio
             onCommitSQL(SQL);
             session.commitAsync();
         } catch (Throwable t) {
-            getLogger().error(t.toString(), t);
+            getLogger().error("", t);
```

Comment:
This should be reverted to preserve existing behavior, as opposed to logging an empty string as the message.